wstesting/wstesting.go

```diff
@@ -398,6 +392,35 @@ type Context struct {
 	sheets map[string]*worksheets.Worksheet
 }
 
+type expr struct {
+	constant worksheets.Value
+	input    string
+}
+
+func (e expr) eval(ctx *Context) (worksheets.Value, error) {
+	// constant
+	if e.constant != nil {
+		return e.constant, nil
+	}
+
+	// lookup
+	ws, ok := ctx.sheets[e.input]
```

Comment:
it's testing so there's a lot more control, but perhaps consider making the lookup explicit to have no possibility of collisions and allow anything to be a literal (members constant/lookup/input).

Reply:
If you name your variables `true`, or `false`, then you could clash, but not otherwise. Maybe we should do the reverse, and forbid any variable that can be a literal?

Reply:
that would be fine, but i'm also ok with moving forward as is